src/components/header/index.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { FC, PropsWithChildren, useMemo } from 'react';
+
+import { Language } from '@smg-automotive/i18n-pkg';
+
+import { Environment } from 'src/types/environment';
+import { Brand } from 'src/types/brand';
+
+import TranslationProvider from '../translationProvider';
+import Stack from '../stack';
+
+import Box from '../box';
+
+import { NavigationLinkProps } from './NavigationLink';
+import { NavigationLanguageMenu } from './NavigationLanguageMenu';
+import { NavigationItems } from './NavigationItems';
+import { NavigationAvatar } from './NavigationAvatar';
+import { useNavigationDrawer } from './hooks/useNavigationDrawer';
+import { NavigationDrawer } from './drawer';
+import { getHeaderLinks } from './config/headerLinks';
+import { DrawerNodeItems, getDrawerNodeItems } from './config/drawerNodeItems';
+import { domains } from './config/domains';
+
+interface NavigationConfiguration {
+  homeUrl: string;
+  currentLanguage: Language;
+  menuHeight: string;
+  user: User | null;
+  drawerNodeItems: DrawerNodeItems;
+  headerLinks: NavigationLinkProps[];
+}
+
+export interface User {
+  id: number;
+  name: string;
+  type: 'private' | 'professional';
+  accountId: number;
+}
+
+export type UserType = 'private' | 'professional';
+export type Plattform = 'as24' | 'ms24';
+
+interface NavigationProps {
+  environment: Environment;
+  plattform: Brand;
+  language: Language;
+  user: User | null;
+  hasNotification: boolean;
+  domain?: string;
+  useAbsoluteUrls?: boolean;
```

Comment:
That prop controls if we are going to use absolute URLs for the navigation links. So when prop is set to `true` the URL is going to look like this `https://www.autoscout24.ch/path`, if we set it to `false` we will have URL like this `/path`. For now we are going to use absolute URLs, because we have to implement header on different domains, but once we have all the apps on same domain we can switch to relative URLs.